pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -505,6 +506,7 @@ private static void start() throws MalformedURLException {
                 responseSerializer
         );
         post(API.USER_ACTIVITY_UPLOADS, new CreateUserActivityUploadRoute(fileUploadService), responseSerializer);
+        get(API.USER_QUESTION_OPTIONS, new GetOptionsForActivityInstanceQuestion(i18nContentRenderer), responseSerializer);
```

Comment:
new route to get REMOTE_AUTOCOMPLETE picklist options. Option labels should be rendered